docker-compose.yml

```
@@ -0,0 +1,25 @@
+---
+version: "3.7"
+
+# This docker-compose file is used to build and test the container
+
+secrets:
+  database_creds:
+    file: ./src/secrets/database_creds.yml
+
+services:
+  trustymail_reporter:
+    # Run the container normally
+    build:
+      context: .
+      dockerfile: Dockerfile
+    depends_on:
+      - redis
+    image: cisagov/trustymail_reporter
```

You almost snuck this one by us! ;)
```suggestion
  pshtt_reporter:
    # Run the container normally
    build:
      context: .
      dockerfile: Dockerfile
    depends_on:
      - redis
    image: cisagov/pshtt_reporter
```